frontend/src/Components/SiteSidebar.tsx

```diff
@@ -36,6 +36,22 @@ export const SiteSidebar = observer((props: SidebarProps) => {
             });
     };
 
+    useEffect(() => {
+        if (!siteInfo) {
+            api.site.updateSiteInfo().then().catch(
+                () => toast.error('Не удалось обновить информацию о сайте!'),
+            );
+        }
+    }, [siteInfo, api.site]);
+
+    useEffect(() => {
+        if (subscriptions === undefined) {
+            api.site.subscriptons().then().catch(
```

Comment:
this is being called multiple times in a row when main page is opened, is this expected?

Reply:
Probably not, lemme check.

Reply:
This effect is called once (can be verified by adding console.log). The `subscription` requests that you see are different (originated from `useFeed`).

There might be a problem with `useFeed`, but it's not a regression introduced in this PR.

Reply:
Please see #235 (a regression from the sorting feature).